### PR TITLE
Added google drive downloading

### PR DESCRIPTION
**Pull request by sbordeyne**
Luckily, I didn't even need google's API, but the requests were a little bit hard to get.

It all works now, though, I've encountered a bug if the download path contains unicode characters (é for instance) I think it's related to the config file saving or something, so I need to investigate, but it's pretty minor, just download to a path without unicode characters inside)

### CODE REVIEW

**Comment by sbordeyne** [comment]
itch.io downloading now works, that was quite the headache to figure out :D

**Comment by sbordeyne** [comment]
just a hiccup though, the header doesn't have the proper filename. It shows up as the linux build, even though it's the windows build

**Comment by sbordeyne** [comment]
well that's because no matter the url, it outputs the linux build hmmm

File: GUI/deprecated/from_itchio.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+lock = Lock()
+
+class DownloadThread(Thread):
+	def __init__(self, get_request, chunks, path, name):
+		Thread.__init__(self)
+		self.get_request = get_request
+		self.chunks = chunks
+		self.path = path
+		if not self.path.endswith("/"):
+			if platform.system().lower()=="windows":
```
Comment (cyian-1756) [inline GUI/deprecated/from_itchio.py:17]: Doesn;t windows allow using / as a path separator?

Comment (sbordeyne) [reply GUI/deprecated/from_itchio.py:17]: no, but I fixed that in the last commit IIRC.

**Pull request merged**